src/models/User.ts

```diff
@@ -80,6 +98,36 @@ const userSchema = new Schema({
     type: String,
     required: true,
   },
+  gender: {
+    type: String,
+    enum: ["MALE", "FEMALE", "OTHERS", null],
+  },
+  age: {
+    type: Number,
+  },
+  address: {
+    type: String,
+  },
+  maritalStatus: {
+    type: String,
+    enum: ["MARRIED", "UNMARRIED", null],
```

Comment:
1. We need these status values
![image](https://github.com/PalisadoesFoundation/talawa-api/assets/16875803/29bed50f-dab0-4c64-9ab9-f8571877bc86)